magstock/templates/registration/new.html

```diff
@@ -0,0 +1,211 @@
+{% extends "base-admin.html" %}
```

Comment:
This is the same thing; it's a copy of the ``new.html`` from core uber, except that we have two new form fields (site label and license plate) and we save those before submitting the check-in form.